LongWarOfTheChosen/Localization/LW_PerkPack_Integrated/XComGame.int

```diff
@@ -689,7 +689,7 @@ LocFriendlyName="Iron Curtain"
 LocFlyOverText="Iron Curtain"
 LocLongDescription="Special shot that does half damage but reduces target mobility for the following two turns. Cone-based attack with primary weapon."
 LocHelpText="Special shot that does low damage but reduces enemy mobility for the following two turns."
-LocPromotionPopupText="<Bullet/> If this cone-based attack hits, Iron Curtain does half normal damage and imposes -5 mobility to the enemy for two turns.<br/><Bullet/> Three-turn cooldown.<br/><Bullet/> Uses three ammo.<br/><Bullet/> Can be used after moving.<br/><Bullet/> Iron Curtain does not work with shotguns or sniper rifles.<br/><Bullet/> Allies in the cone can be hit by this ability.<br/>"
+LocPromotionPopupText="<Bullet/> If this cone-based attack hits, Iron Curtain does half normal damage and imposes -5 mobility to the enemy for two turns.<br/><Bullet/> Iron Curtain has a <Ability:SelfCooldown_LW/> turn cooldown.<br/><Bullet/> Uses three ammo.<br/><Bullet/> Can be used after moving.<br/><Bullet/> Iron Curtain does not work with shotguns or sniper rifles.<br/><Bullet/> Allies in the cone can be hit by this ability.<br/>"
```

Comment:
I don't think it's necessary to repeat the name of the ability each time. A simple:
> * Has an N-turn cooldown

would work fine

Reply:
The reason for this particular template for cooldown line is that "<Name> has a <Ability:SelfCooldown/> turn cooldown" is the most common way used by vanilla descriptions. Also, it does not use hyphen before "turn". Which is why I chose it. Personally, I don't mind it either way, but I think at least it should be as consistent as possible.

Reply:
You mean vanilla is verbose and grammatically wrong? :smile: Well, if you're sure. I generally find that vanilla is inconsistent with itself. But if most of the abilities are like that, then OK, let's go with it.